_layouts/post.html

```diff
@@ -31,6 +31,27 @@ <h1 class="post-title heading" itemprop="name headline">{{ page.title }}</h1>
           </p>
         </div>
       </div>
+      <div class="event-add">
+        {% capture event_title %}{{ page.title }}{% endcapture %}
+        {% capture event_location %}{{ page.location }}{% endcapture %}
+        {% capture event_description %}{{ page.text|strip }} For details, link here: {{ page.url|strip | prepend: site.url }}{% endcapture %}
+        <script type="text/javascript">
+          var event_data = {
+            LOCATION:'{{ event_location }}',
+            SUMMARY:'{{ event_title }}',
+            DESCRIPTION:'{{ event_description }}',
+            DTSTART:'{{ page.date | date: "%-m/%-d/%Y" }} {{ page.startTime | date: "%-I:%M %P" }}',
```

Comment:
This is a different format that `ics.js` expects. I'll see if I can get it to use the Android one—should probably just have one format for all these ICS stuff.